apps/server/src/models/targets.ts

```diff
@@ -113,10 +104,4 @@ Targets.init(
   }
 )
 
-Targets.belongsTo(TargetWallets, {
-  foreignKey: 'walletId',
-  as: 'wallet',
-  onDelete: 'CASCADE',
-})
-
```

Comment:
well its a one to one relationship we have between the targets and the targetWallet. Since we moved the Fk from the target to the targetWallet. i.e  we deleted the walletId in target model and introduced targetId in the wallet model. hence writing the association as above becomes wrong. 
Based on our model design, TargetWallet belongsTo Targets.

Reply:
Also I ended working on the targets endpoint because we made mistake in  how the tasks were assigned. my task is to update all the wallet models(user, group and target) and the affected endpoints. and then there were still tickets to updetate target model, user model and group models separately. what we should have done was assign everything concerning users to one person, groups to one person and targets to another person.